### DISCUSSION TITLE
Problem with micrometer-binders

### DISCUSSION THREAD

**Discussion by tomazfernandes** [spring-projects/spring-kafka#2231] [General]
It seems we're having some issue with this dependency:

```
Execution failed for task ':spring-kafka:compileKotlin'.
> Error while evaluating property 'filteredArgumentsMap' of task ':spring-kafka:compileKotlin'
   > Could not resolve all files for configuration ':spring-kafka:compileClasspath'.
      > Could not find io.micrometer:micrometer-binders:.
        Required by:
            project :spring-kafka
```

Since we've had issues with `micrometer` before, I'm assuming it's something they've changed and not something local from my setup.

Thanks!

**Comment by tomazfernandes** [comment]
I saw Andy Wilkinson just opened a PR for this, and made the necessary adjustments locally, so it seems it's all good. Thanks.

**Reply by garyrussell** [reply]
Merged; please rebase.
